src/IridiumSBD.h

```diff
@@ -177,6 +180,7 @@ class IridiumSBD
       wireport = &wirePort;
       deviceaddress = deviceAddress;
       sbdixInterval = ISBD_USB_SBDIX_INTERVAL;
+      sbdSessionTimeout = 0;
```

Comment:
Same comment, I suggest replacing `0` with `ISBD_DEFAULT_SBDSESSION_TIMEOUT`

Reply:
Changed as suggested, thank you